### PR TITLE
Subregion code

### PR DESCRIPTION
**Pull request by evasinha**
Code modified to read value for Snake River Basin subregion.

### CODE REVIEW

File: R/setup.R [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #' be for the same region and scenario type as the \code{aScenarioInfo} object.
 #' @author KVC October 2017
 #' @export
-LandAllocator_setup <- function(aLandAllocator, aScenarioInfo, agData=NULL) {
+LandAllocator_setup <- function(aLandAllocator, aScenarioInfo, agData=NULL, subregion=NULL) {
```
Comment (rplzzz) [inline R/setup.R]: It looks like you did not document the subregion argument.  Also, other than testing for whether it's `NULL` or not, the subregion argument doesn't look like it's ever used.  So, it's really more like a flag that we test to see whether or not we want to use the alternate dataset.
